src/applications/disability-benefits/view-payments/components/view-payments-lists/payments/Payments.jsx

```diff
@@ -27,21 +27,19 @@ const Payments = ({
   textContent,
   alertMessage,
 }) => {
+  // Page navigation
   const location = useLocation();
   const navigate = useNavigate();
-  const [currentData, setCurrentData] = useState([]);
   const currentPage = new URLSearchParams(location.search).get('page') || 1;
-  const totalPages = useRef(0);
-  const tableHeadingRef = useRef(null);
 
-  useEffect(
-    () => {
-      const paginatedData = paginateData(data);
-      setCurrentData(paginatedData[currentPage - 1]);
-      totalPages.current = paginatedData.length;
-    },
-    [currentPage, data],
-  );
+  // State
+  const [currentData, setCurrentData] = useState([]);
+  const [maxPageLength, setMaxPageLength] = useState(10);
```

Comment:
If we move the small/large screen page length numbers up to `BIG_CONSTANTS` at the top, we can use that here as well

Reply:
@dysbo I'm not quite sure I understand your suggestion. I can't move this local state out of the component because React doesn't allow hooks to run outside the functional component. LMK if I'm missing the intent; happy to refactor as needed!